src/cnc/flavors/aws/lambda-lite/1/provision/partials/collection/providers.tf.j2

```diff
@@ -0,0 +1,40 @@
+provider "aws" {
```

Comment:
nit: I think we can make this a shared template if it's the same as the one from the ecs flavor

Reply:
How could we do this?